### ISSUE TITLE
Admin interface content standardization

### ISSUE THREAD

**Issue by phylophyl** [freedombox/Plinth#442]
/plinth/apps/ikiwiki/ is a good exemple to start having a systematic content structure for each app page in Plinth. 

What is it?
What to do step by step after activation to administrate the service
How to share with friends and family

**Comment by harry-7** [comment]
I would like to work on this Issue. By Each App page do You mean index of each app right ?

**Comment by jvalleroy** [comment]
@harry-7 Yes, for now at least, we can add this info to the index page for each app.
